ui/src/app/app.constants.ts

```diff
@@ -84,6 +84,26 @@ export const Profile = {
   ExistingConnProfile: 'existing',
 }
 
+export const Dialect = {
+  PostgreSQLDialect: 'experimental_postgres'
+}
+
+export const PGSQLDatatype = {
+  Bytea: 'BYTEA',
+  Float8: 'FLOAT8',
+  Int8: 'INT8',
+  Varchar: 'VARCHAR',
+  TimestampTZ: 'TIMESTAMPTZ'
+}
+
+export const GoogleSQLDatatype = {
+  Bytes: 'BYTES',
+  Float64: 'FLOAT64',
+  Int64: 'INT64',
+  String: 'STRING',
+  Timestamp: 'TIMESTAMP'
+}
+
```

Comment:
On second thought, isn't this `application data`? Shouldn't we be getting this data via an API call from the backend instead of hardcoding it here?

Reply:
Moved these to API call